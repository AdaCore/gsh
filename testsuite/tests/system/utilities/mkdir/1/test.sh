echo "create simple directories"
mkdir a b
echo $?

echo "create recursively a directory"
mkdir -p a/b/c a 
echo $?

echo "check if error is raised"
mkdir a b
echo $?

echo "check that no error is raised with -p"
mkdir -p a b
echo $?

echo "check creation of directories with spaces"
mkdir -p " b a"/b
echo $?
cd " b a"/b
touch toto.txt
cd ../..
ls */*/toto.txt

true
