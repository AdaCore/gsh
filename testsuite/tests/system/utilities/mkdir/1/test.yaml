title: mkdir command - builtin -
driver: shell_script
