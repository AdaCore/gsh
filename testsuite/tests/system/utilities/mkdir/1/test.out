create simple directories
0
create recursively a directory
0
check if error is raised
cannot create <TEST_DIR>/a
cannot create <TEST_DIR>/b
1
check that no error is raised with -p
0
check creation of directories with spaces
0
 b a/b/toto.txt
